gateway-provider-security-jwt/src/main/java/org/apache/knox/gateway/provider/federation/jwt/filter/JWTAuthCodeAssertionFilter.java

```diff
@@ -55,6 +57,10 @@ public void init( FilterConfig filterConfig ) throws ServletException {
     GatewayServices services = (GatewayServices) filterConfig.getServletContext().getAttribute(GatewayServices.GATEWAY_SERVICES_ATTRIBUTE);
     authority = services.getService(ServiceType.TOKEN_SERVICE);
     sr = services.getService(ServiceType.SERVICE_REGISTRY_SERVICE);
+
+    this.tokenIssuer = filterConfig.getInitParameter(JWTAccessTokenAssertionFilter.ISSUER) != null
+            ? filterConfig.getInitParameter(JWTAccessTokenAssertionFilter.ISSUER)
```

Comment:
As per RFC https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.1 issuer is a `principal` and not principals but could there be cases where we might need to support multiple issuers? if not we are golden. Just wanted to raise this.

Reply:
I don't know how likely this is, but the doc says it's a StringOrURI not an array. If we ever need to support multiple issuers we can still use coma separated values I guess, or introduce new application specific claim which is an array.

Reply:
I can't imagine a case for multiple issuers. In that case, we would use multiple topologies IMO.